packages/manager/apps/container/src/cookie-policy/CookiePolicy.tsx

```diff
@@ -78,7 +78,7 @@ const CookiePolicy = ({ shell }: Props): JSX.Element => {
       animation={false}
     >
       <Modal.Body>
-        <div className="cookie-policy-modal p-1">
+        <div className="cookie-policy-modal-description p-1">
```

Comment:
**nitpick:** You also need to change the class name in css/scss file no?

Reply:
I didn't find this class in css / scss

Reply:
`packages/manager/modules/cookie-policy/src/style.scss` Line 8 ?

Reply:
I don't think this is used by container. Container cookie modal style come from `packages/manager/apps/container/src/cookie-policy/cookie-modal.module.scss` I don't think the react stack is using old angularjs module scss file

Reply:
or maybe it does but if it does, is it normal ?

Reply:
but to be be more precise, the react component is using the packages/manager/modules/cookie-policy/src/style.scss style file which is the same as packages/manager/modules/cookie-policy/src/style.scss without the class selector